index.html

```diff
@@ -654,5 +654,400 @@ <h2>Exposing change of MediaStreamTrack configuration</h2>
     </p>
     </div>
   </section>
+  <section>
+    <h2>Human faces</h2>
+    <p>Human face metadata describes the human faces in video frames. It can
+    be set by web applications using the standard means when creating
+    {{VideoFrameMetadata}} for {{VideoFrame}}s or it can be set by an user agent
+    when the media track constraints, defined below, are used to
+    enable face detection for the {{MediaStreamTrack}} which provides the
+    {{VideoFrame}}s.</p>
+    <p>The facial metadata may be used by video encoders to enhance the quality
+    of the faces in encoded video streams or for other suitable purposes.</p>
+    <section>
+      <h3>{{VideoFrameMetadata}}</h3>
+      <pre class="idl">
+partial dictionary VideoFrameMetadata {
+  sequence&lt;HumanFace&gt; humanFaces;
+};</pre>
+      <section class="notoc">
+        <h4>Members</h4>
+        <dl class="dictionary-members" data-link-for="VideoFrameMetadata" data-dfn-for="VideoFrameMetadata">
+          <dt><dfn><code>humanFaces</code></dfn> of type <span class="idlMemberType"><code>sequence&lt;{{HumanFace}}&gt;</code></span></dt>
+          <dd>
+            <p>The set of known human faces in this video frame.</p>
+          </dd>
+        </dl>
+      </section>
+    </section>
+    <section>
+      <h3>{{HumanFace}}</h3>
+      <pre class="idl">
+dictionary HumanFace {
+  long              id;
+  float             probability;
+  DOMRectReadOnly   boundingBox;
+  HumanFaceLandmark leftEye;
+  HumanFaceLandmark rightEye;
+  HumanFaceLandmark mouth;
+};</pre>
+      <section class="notoc">
+        <h4>Dictionary {{HumanFace}} Members</h4>
+        <dl class="dictionary-members" data-dfn-for="HumanFace" data-link-for="HumanFace">
+          <dt><dfn><code>id</code></dfn> of type <span class="idlMemberType">{{long}}</span></dt>
+          <dd>
+            <p>If set, an unique identifer of a face within a sequence. If the same face can be tracked in multiple
+            frames originating from the same {{MediaStreamTrack}} source, {{id}} is set to the same integer value
+            for the face in all frames.</p>
+            <p>User agent MUST NOT select the assigned value of {{id}} in such a way that the detected faces could
+            be correlated to match in any way between different {{MediaStreamTrack}} objects.</p>
+          </dd>
+          <dt><dfn><code>probability</code></dfn> of type <span class="idlMemberType">{{float}}</span></dt>
+          <dd>
+            <p>If set, a probability value in range (0,1].
+            The approximate probability of the {{HumanFace}} specifying an actual human face.</p>
```

Comment:
In addition, range notation can be confusing to read, so I'd suggest explaining the range of allowed values in full: If set, a probability value greater than zero and with a maximum of 1.

Reply:
@jan-ivar By the commonly used definition of probability, it is always zero or greater with a maximum of one. Since it doesn't make sense to return zero as a probability, it could be used as a special value.

I updated the specific definition of `probability` in the spec, hopefully it is more precise now. Quoting the latest version:
 "[probability] is the estimate of the conditional probability that the segmented object actually is of the type indicated by the {{Segment/type}} member on the condition that the detection has been made." Note that it would be an estimate of the probability, not probability itself which is usually impossible to exactly calculate.

An implementer could estimate the probability by feeding some test video into the face detection algorithm and manually verifying how large percentage of the detections are correct (if the algorithm outputs a "score", "confidence" or similar, this could be taken also into account). 

A probability estimate of `1` would likely be inaccurate for non-synthetic video, but I think for a highly accurate algorithm it might make sense to return it (close enough to `1`). I think generally `1` definitely should not be a common value. Typically platform face detection implementation are not that accurate. 

@chrisn Changed the range notation.